src/main/resources/database/coreSchema.sql

```diff
@@ -60,8 +61,8 @@ CREATE  TABLE IF NOT EXISTS `core`.`modules` (
   `moduleType` VARCHAR(16) NOT NULL ,
   `moduleCategory` VARCHAR(64) NULL ,
   `moduleCategoryLangPointer` VARCHAR(64) NULL ,
-  `moduleResult` VARCHAR(256) NULL ,
-  `moduleHash` VARCHAR(256) NULL UNIQUE,
+  `moduleResult` VARCHAR(191) NULL ,
+  `moduleHash` VARCHAR(191) NULL UNIQUE,

```

Comment:
Why the reduction in memory?

Reply:
UTF-8 encoding requires 191 because four-byte chars on mysql 5.5 can't fit if mem > 191.